contracts/ForwarderV4.sol

```diff
@@ -372,7 +372,7 @@ contract ForwarderV4 is IERC721Receiver, ERC1155Receiver, IForwarderV4 {
     public
     view
     virtual
-    override(ERC1155Receiver, IERC165)
+    override(ERC1155Holder, IERC165)
```

Comment:
When override is used without any parentheses, it means that the function overrides a function from a single unspecified base contract. Solidity will look through all the base contracts to find a function that matches the name and parameters of the overriding function.

When override(ERC1155Holder, IERC165) is used, it means that the function overrides functions from the specified base contracts ERC1155Holder and IERC165. This is used when multiple base contracts have functions with the same name and parameters, and you want to specify which ones are being overridden. This is a requirement since Solidity 0.6.0 to avoid ambiguity.